src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -129,9 +61,9 @@ const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
         <Form isWidthLimited>
           <FormGroup
             label="Name"
-            helperTextInvalid={nameHelperTextInvalid}
-            validated={nameValidated}
+            helperTextInvalid={errors.satelliteManifestName?.message}
             helperTextInvalidIcon={<ExclamationCircleIcon />}
+            validated={errors.satelliteManifestName ? 'error' : 'noval'}
```

Comment:
It looks like patternfly suggests "default" for the... default state.

Reply:
Just updated.